src/languages/gohtml.coffee

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+
+  name: "Golang Template"
+  description: "Comma-Separated Values"
+  namespace: "gohtml"
+
```

Comment:
Consider adding:

```
fallback: ["html"]
```

So `html` options will be used if `gohtml` is not set explicitly.